### PR TITLE
50 suspress chmod warning

### PR DESCRIPTION
**Pull request by gusdecool**
Issue: https://github.com/schmittjoh/metadata/issues/50

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 vendor
+.idea/
```
Comment (aequasi) [inline .gitignore]: ide project folders should probably be kept in your global gitignore, FWIW

**Comment by aequasi** [comment]
@schmittjoh can we get this committed and tagged? The work-around is annoying.

**Comment by aderuwe** [comment]
@schmittjoh I really need this too (see https://github.com/symfony/symfony/issues/2125 too)

@gusdecool can you please remove the .gitignore part of this PR? It has no business here

**Comment by gusdecool** [comment]
@aderuwe thanks for the notice. sorry took time. was very busy this month.

PR update :+1:

**Comment by vlad-ko** [comment]
+ 1 for this. for those using vagrant it's a big deal.

**Comment by e-moe** [comment]
:+1: looking forward for merge

**Comment by aderuwe** [comment]
@schmittjoh please could we have this merged and tagged?

**Comment by e-moe** [comment]
:+1:

**Comment by gusdecool** [comment]
@schmittjoh any info if this PR accepted or not?

**Comment by aequasi** [comment]
@schmittjoh Is there any reason this isn't being merged/tagged?

**Comment by schmittjoh** [comment]
Thanks, it still looks a bit like a hack, but we also do not have any good alternatives.

**Comment by aequasi** [comment]
Agreed. Will have to keep an eye on symfony to see if they come up with a better solution

**Pull request merged**